cocos/2d/CCFontAtlas.cpp

```diff
@@ -413,9 +419,8 @@ bool FontAtlas::prepareLetterDefinitions(const std::u32string& utf32Text)
                     {
                         tex->setAliasTexParameters();
                     }
-                    tex->initWithData(_currentPageData, _currentPageDataSize,
-                        pixelFormat, CacheTextureWidth, CacheTextureHeight, Size(CacheTextureWidth, CacheTextureHeight));
```

Comment:
The underline texture format should be `RGBA8888` or `A8`,  here the `pixelFormat` is `AI88` or `A8`.

Reply:
AI88 does not support in metal, so I think should change it to RGBA8888

Reply:
yes, the format is mapped in `initTextureWithZeros`